src/Native/Unix/System.Native/pal_mount.c

```diff
@@ -134,13 +139,14 @@ SystemNative_GetFormatInfoForMountPoint(const char* name, char* formatNameBuffer
         }
         else
         {
-            SafeStringCopy(formatNameBuffer, bufferLength, stats.f_fstypename);
+            SafeStringCopy(formatNameBuffer, Int32ToSizeT(bufferLength), stats.f_fstypename);
             *formatType = -1;
         }
 #else
-        assert(formatType != nullptr);
-        *formatType = SignedCast(stats.f_type);
-        SafeStringCopy(formatNameBuffer, bufferLength, "");
+        assert(formatType != NULL);
+        assert(stats.f_type <= INT_MAX);
```

Comment:
Why do we need this assert comparing to INT_MAX (which is 32 bits) when we cast it to int64_t below? Should it rather be INT64_MAX? And it seems we should add casting function for it for the sake of consistency.

Reply:
`f_type` is `unsigned long` in alpine and `int`. I have just removed the assert and kept the cast inline.